src/utils/rangeHealthFactor.ts

```diff
@@ -0,0 +1,16 @@
+export const getRangeHealthFactor = (
+  fixedLow: number,
+  fixedHigh: number,
+  currentAPR: number,
+): 1 | 2 | 3 => {
```

Comment:
can you add this as type? Perhaps: type HealthFactor = 1 | 2 | 3; Like this it is really unclear what 1,2,3 mean.

Reply:
Done.